AppCommonNetworking/XNetworking/src/commonMain/kotlin/jp/co/soramitsu/xnetworking/txhistory/client/sorawallet/SubQueryClientForSoraWallet.kt

```
@@ -59,8 +59,8 @@ class SubQueryClientForSoraWallet(
         address: String,
         page: Long,
         filter: ((TxHistoryItem) -> Boolean)? = null
-    ): TxHistoryResult<TxHistoryItem> {
-        val explorerUrl = soraRemoteConfigBuilder.getConfig().blockExplorerUrl
+    ): TxHistoryResult<TxHistoryItem>? {
```

Can we throw exception in result instead of returning nullable result? 
I suppose it will be uncomfortable to work with nullable result